libqtile/extension/window_list.py

```diff
@@ -48,7 +48,7 @@ def list_windows(self):
         for win in windows:
             if win.group:
                 item = self.item_format.format(
-                    group=win.group.name, id=id, window=win.name)
+                    group=win.group.label or win.group.name, id=id, window=win.name)
```

Comment:
Yes, but the label may also be the empty string(!)
in that case, the group is not dislayed in GroupBox widget(, and that is my initial requirement.)
However, for WindowList the windows are listed and belong to a group.
If the group has an empty label, the name of the group is used in formatting.

In Group.__init__ we have three possible values for label:
- label = None (default) --> not set, use the group name as label
- label = <empty string> --> the label is set explicitly to an empty string (hidden in GroupBox)
- label = <text> --> a meaningfull text to be displayed